src/components/text-editor/text-editor.tsx

```diff
@@ -105,6 +110,9 @@ export class TextEditor implements FormComponent<string> {
     @Prop()
     public plugins: CustomElement[] = [];
```

Comment:
Not part of this PR, but I think we should rename this to something like `customElements` instead. Calling it `plugins` is a bit confusing for the consumer

Reply:
100% agree. This is the plan as well